exporter/elasticsearchexporter/integrationtest/go.mod

```diff
@@ -255,3 +255,5 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/sapmr
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry => ../../../pkg/resourcetotelemetry
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter => ../../prometheusremotewriteexporter
+
+replace github.com/open-telemetry/opentelemetry-collector-contrib/extension/ackextension => ../../../extension/ackextension
```

Comment:
I can't figure why this change is there. Is it added by `make multimod`?

Reply:
or `make crosslink` ?

Reply:
it looks like `make crosslink` is adding this line